docs/advanced-topics/code_coverage.md

```diff
@@ -7,53 +7,50 @@ permalink: /advanced-topics/code-coverage/
 ---
 
 # Code Coverage
+{: .no_toc}
 
 You can generate code coverage report for your project using [Clang Source-based
-Code Coverage].
+Code Coverage]. This page walks you through common issues in generating reports.
```

Comment:
Do you think this sentence is nice to have? For me it sets up some negative context as it warns about the issues I'm going to face :)

Reply:
I was trying to give some context for what the page contains, but wasn't sure if the steps in the page relate to the instructions on the linked Clang page. Are they part of the same process, or different processes?

Reply:
The Clang page linked here is basically a reference to the toolchain we're using for generating code coverage reports. The readers of the OSS-Fuzz page do not really need to open / read that Clang page. This OSS-Fuzz page is the instruction on how to generate code coverage report for a project integrated with OSS-Fuzz.

Answering your question, it's the same process, but Clang docs cover lower level details (not necessary to know), while our page gives a higher level how-to.

Reply:
Ah, got it! Rephrased a bit to make that clear.